supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -21,6 +21,28 @@
 
 *See also*:
 
+[discrete]
+[[network-interface-card]]
+==== image:images/no.png[no] network interface card
+*Description*: A dispreferred referent for the acronym "NIC". Use "network interface controller" instead.
```

Comment:
"dispreferred reference" doesn't roll of the tongue for me. Suggest something simpler or rewording. Maybe:

`Do not use "network interface card" for the acronym "NIC". Use "network interface controller" instead.`
or
`Do not spell out the acronym "NIC" as "network interface card". Use "network interface controller" instead.`
or
`An alternative meaning for the acronym "NIC". Use "network interface controller" instead.`

Reply:
Maybe "*Description*: Substitute "network interface controller instead"."

Reply:
Agreed and changed